src/r1/interval.rs

```diff
@@ -169,6 +169,19 @@ impl Interval {
             (self.lo - other.lo).abs() <= EPSILON && (self.hi - other.hi).abs() <= EPSILON
         }
     }
+
+    /// Returns the Hausdorff distance to the given interval.
+    /// For two intervals x and y, this distance is defined as
+    /// h(x, y) = max_{p in x} min_{q in y} d(p, q).
+    pub fn directed_hausdorff_distance(&self, other: &Self) -> f64 {
+        if self.is_empty() {
+            return 0.0
```

Comment:
Could you format the changes? It seems that the code is not `cargo fmt`tted.

```diff
diff --git a/src/r1/interval.rs b/src/r1/interval.rs
index 4d46f7b..e12e82b 100644
--- a/src/r1/interval.rs
+++ b/src/r1/interval.rs
@@ -175,12 +175,12 @@ impl Interval {
     /// h(x, y) = max_{p in x} min_{q in y} d(p, q).
     pub fn directed_hausdorff_distance(&self, other: &Self) -> f64 {
         if self.is_empty() {
-            return 0.0
+            return 0.0;
         }
         if other.is_empty() {
             return f64::INFINITY;
         }
-        0_f64.max((self.hi-other.hi).max(other.lo-self.lo))
+        0_f64.max((self.hi - other.hi).max(other.lo - self.lo))
     }
 }
 ```

Reply:
Done. Sorry about that.